backend/src/JsonTranslator.py

```diff
@@ -0,0 +1,112 @@
+from flask import request
+import sys
+sys.path.append("..")
+
+
+
+# param: Json, cour
+# return: Json
+def searchQuery(cur):
```

Comment:
There is no signature definition of this function, we should create a standard structure based on some style (not only for python, but for all languages)